### PR TITLE
{bio}[GCC/11.3.0] CD-HIT v4.8.1, GenomeTools v1.6.2, Kent_tools v442, ...

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

depends on https://github.com/easybuilders/easybuild-easyconfigs/pull/17245

### CODE REVIEW

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 14 out of 14 (8 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/14ab0b6eeec8652e105412fab2d2cc0c for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 15 out of 17 (8 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux Ubuntu 20.04.2 LTS (Focal Fossa), x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/4ee08fe4b11c285518db4c1fc65a1c2b for a full test report.

File: easybuild/easyconfigs/r/RepeatModeler/RepeatModeler-2.0.4-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    'PATH': '',
+    'PERL5LIB': '',
+}
+
+sanity_check_paths = {
+    'files': _bins + ['Job.pm', 'RepeatUtil.pm', 'RepModelConfig.pm'],
+    'dirs': ['Libraries', 'Matrices', 'util'],
+}
+
+sanity_check_commands = ["RepeatModeler -help | grep 'RepeatModeler - Model repetitive DNA'"]
```
Comment (branfosj) [inline easybuild/easyconfigs/r/RepeatModeler/RepeatModeler-2.0.4-foss-2022a.eb:65]:
```
== 2023-02-03 09:05:57,346 build_log.py:171 ERROR EasyBuild crashed with an error (at easybuild/easybuild-framework/easybuild/base/exceptions.py:126 in __init__): Sanity check failed: sanity check command RepeatModeler -help | grep 'RepeatModeler - Model repetitive DNA' exited with code 1 (output: Can't locate Time/HiRes.pm in @INC (you may need to install the Time::HiRes module) (@INC contains: /project/60006/boegelbot/Rocky8/zen2/software/RepeatModeler/2.0.4-foss-2022a /project/def-maintainers/boegelbot/Rocky8/zen2/software/RepeatModeler/2.0.4-foss-2022a /project/def-maintainers/boegelbot/Rocky8/zen2/software/RepeatMasker/4.1.4-foss-2022a /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5) at /project/def-maintainers/boegelbot/Rocky8/zen2/software/RepeatModeler/2.0.4-foss-2022a/RepeatModeler line 136.
BEGIN failed--compilation aborted at /project/def-maintainers/boegelbot/Rocky8/zen2/software/RepeatModeler/2.0.4-foss-2022a/RepeatModeler line 136.
) (at easybuild/easybuild-framework/easybuild/framework/easyblock.py:3590 in _sanity_check_step)
```

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/k/Kent_tools/Kent_tools-442-GCC-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    ('MariaDB', '10.9.3'),
+    ('libpng', '1.6.37'),
+    ('zlib', '1.2.12'),
+    ('util-linux', '2.38'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+    ('freetype', '2.12.1'),
+]
+
+buildopts = 'CC="$CC" COPT="$CFLAGS -fcommon" PNGLIB="-L$EBROOTLIBPNG/lib -lpng" ZLIB="-L$EBROOTZLIB/lib -lz" '
+buildopts += 'SSL_DIR="$EBROOTOPENSSL" SSLDIR="$EBROOTOPENSSL" MYSQLLIBS="-L$EBROOTMARIADB/lib -lmariadb -lstdc++" '
```
[the remark this answers is not in the document]
Comment (branfosj) [reply easybuild/easyconfigs/k/Kent_tools/Kent_tools-442-GCC-11.3.0.eb:31]: Adding
```
prebuildopts = 'sed -i "s/rsync -a /cp /" %(builddir)s/userApps/kent/src/parasol/makefile && '
```
fixes this issue.

File: easybuild/easyconfigs/r/RepeatModeler/RepeatModeler-2.0.4-foss-2022a.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/r/RepeatModeler/RepeatModeler-2.0.4-foss-2022a.eb:65]: something looks very wrong with Perl in `jsc-zen2`
from the @INC path in the error it uses system Perl although Perl is listed as dep

`@INC contains: /project/60006/boegelbot/Rocky8/zen2/software/RepeatModeler/2.0.4-foss-2022a /project/def-maintainers/boegelbot/Rocky8/zen2/software/RepeatModeler/2.0.4-foss-2022a /project/def-maintainers/boegelbot/Rocky8/zen2/software/RepeatMasker/4.1.4-foss-2022a /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5
`

Comment (smoors) [reply easybuild/easyconfigs/r/RepeatModeler/RepeatModeler-2.0.4-foss-2022a.eb:66]: the problem was the shebang for the RepeatModeler executable was not fixed yet

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/755e9dddcb1df7324ca918ccfa1e5a05 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux Ubuntu 20.04.2 LTS (Focal Fossa), x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/0c4c1a7a73159aed9be78420f519e870 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @smoors!

**Pull request merged**